wezterm-mux-server-impl/src/sessionhandler.rs

```diff
@@ -795,3 +811,64 @@ async fn domain_spawn_v2(spawn: SpawnV2, sender: PduSender) -> anyhow::Result<Pd
         size: tab.get_size(),
     }))
 }
+
+async fn domain_spawn_popup(spawn_popup: SpawnPopup, sender: PduSender) -> anyhow::Result<Pdu> {
```

Comment:
This is a copy and paste of `domain_spawn` above for the moment to get build working as I work my way through the understanding of this code.  As I start implementation next of the cli command, this more than likely will change, and if it doesn't, I'll just reuse the previous implementation.